### PR TITLE
docs(orchestrator): fix stale links

### PR DESCRIPTION
**Pull request by batzionb**
backport https://github.com/janus-idp/backstage-plugins/pull/2551

### CODE REVIEW

File: plugins/orchestrator/README.md [hunk 1]
```diff
@@ -156,54 +156,55 @@ For more information about the configuration options, including other optional p
...
-> **NOTE:**\
-> While the OpenAPI specification is available in the Orchestrator plugin, the UI currently does not rely on this spec. \
-> We plan to incorporate v2 endpoints into the UI in the near future.
+OpenAPI specification [file](https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/orchestrator-common/src/openapi/openapi.yaml) is available in [orchestrator-common](https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/orchestrator-common).  
+OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/orchestrator-common/src/generated/docs/markdown/README.md)
 
 ### orchestrator-common
 
-The typescript schema is generated in [auto-generated](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/auto-generated/api/models/schema.ts) folder from openapi.yaml specification file.
+The typescript client is generated in [auto-generated](https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/orchestrator-common/src/generated/client) folder from openapi.yaml specification file.
```
Comment (gciavarrini) [inline plugins/orchestrator/README.md:180]: `s/[auto-generated]/[generated]/

File: plugins/orchestrator/README.md [hunk 2]
```diff
@@ -156,54 +156,55 @@ For more information about the configuration options, including other optional p
...
 
 `git clone git@github.com:janus-idp/backstage-plugins.git`
 
-If you need to change the OpenAPI spec, edit the [openapi.yaml](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/openapi/openapi.yaml) according to your needs and then execute from the project root folder:
+If you need to change the OpenAPI spec, edit the [openapi.yaml](https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/orchestrator-common/src/openapi/openapi.yaml) according to your needs and then execute from the project root folder:
 
 `yarn --cwd plugins/orchestrator-common openapi`
 
-This command updates the [auto-generated files](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/auto-generated/api/) and the [auto-generated docs](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/orchestrator-common/src/auto-generated/docs).
+This command updates the [auto-generated files](https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/orchestrator-common/src/generated) including API, client and docs.
```
Comment (gciavarrini) [inline plugins/orchestrator/README.md:203]: `s/[auto-generated files]/[generated files]/

**Review by gciavarrini — CHANGES_REQUESTED** [review]

File: plugins/orchestrator/README.md (hunk 2 above)
Comment (batzionb) [reply plugins/orchestrator/README.md]: fixed

**Review by gciavarrini — APPROVED** [review]
/lgtm

**Pull request merged**